IFC4x3/Properties/h/HollowSleeperPurposeOfUsed_24bWI7gefEm81rwgNZrJim/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="HollowSleeperPurposeOfUsed_24bWI7gefEm81rwgNZrJim" Name="HollowSleeperPurposeOfUsed" UniqueId="84960487-aa8a-4ec0-8075-eaa5e3d53b30" PrimaryDataType="IfcLabel" />
```

Comment:
Enum?

Reply:
1. Exising Property from Track Elements use: `HollowSleeperUsage_24bWI7gefEm81rwgNZrJim`
2. Enum might be an idea if a finite list can be made.